### ISSUE TITLE
DevContainer Template not `as_json`able when pretend running `rails new testapp --devcontainer --pretend`

### ISSUE THREAD

**Issue by plattenschieber** [rails/rails#53441]
### Steps to reproduce

```bash
rails new testapp --devcontainer --pretend
```

### Expected behavior

It should run through as it does when leaving out the `--pretend` flag.

### Root cause?!
I've been debugging through the code, but don't understand what's different in the `--pretend` mode when rails is [templating the `devcontainer.json.tt`](https://github.com/rails/rails/blob/7fac5d14d00fcc8dc60b49729abe78223ad3bf00/railties/lib/rails/generators/rails/devcontainer/devcontainer_generator.rb#L37):
```ruby
template "devcontainer/devcontainer.json", ".devcontainer/devcontainer.json"
```
but somehow in `--pretend` mode, one of the features seems to be a hash and raises `block in template': undefined method 'as_json' for an instance of Hash (NoMethodError)` when [tried to be `as_json`ed](https://github.com/rails/rails/blob/7fac5d14d00fcc8dc60b49729abe78223ad3bf00/railties/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt#L11)
```ruby
<%= features.map { |key, value| "\"#{key}\": #{value.as_json}" }.join(",\n    ") %>
```
### Possibly mitigation
I've changed the raising method to `to_json` and added a `require 'json'` to the fail, which solved the issue:

```diff
diff --git a/devcontainer.json.tt b/devcontainer.json.tt
index 55eba9d..47dd156 100644
--- a/devcontainer.json.tt
+++ b/devcontainer.json.tt
@@ -1,3 +1,4 @@
+<% require 'json' %>
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/ruby
 {
@@ -8,7 +9,7 @@
 
   // Features to add to the dev container. More info: https://containers.dev/features.
   "features": {
-    <%= features.map { |key, value| "\"#{key}\": #{value.as_json}" }.join(",\n    ") %>
+    <%= features.map { |key, value| "\"#{key}\": #{value.to_json}" }.join(",\n    ") %>
   },
 
 <%- if !container_env.empty? -%>
@@ -18,7 +19,7 @@
 <%- end -%>
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": <%= forward_ports.as_json %>,
+  "forwardPorts": <%= forward_ports.to_json %>,
 
   // Configure tool-specific properties.
   // "customizations": {},
```

### Actual behavior

```bash
/workspaces/test🔒 via 💎 v3.3.4 
⬢ [Docker] ❯ bundle exec rails --version
Rails 7.2.1.2

/workspaces/test🔒 via 💎 v3.3.4 
⬢ [Docker] ❯ bundle exec rails new testapp --devcontainer -p
      create  
      create  README.md
      [...]
      create  .devcontainer
bundler: failed to load command: rails (/home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/bin/rails)
/home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in `block in template': undefined method `as_json' for an instance of Hash (NoMethodError)
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in `each'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in `map'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in `template'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/3.3.0/erb.rb:429:in `eval'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/3.3.0/erb.rb:429:in `result'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/file_manipulation.rb:128:in `block in template'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/create_file.rb:54:in `render'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/create_file.rb:64:in `block (2 levels) in invoke!'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/create_file.rb:64:in `open'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/create_file.rb:64:in `block in invoke!'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/empty_directory.rb:117:in `invoke_with_conflict_check'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/create_file.rb:61:in `invoke!'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions.rb:93:in `action'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/create_file.rb:25:in `create_file'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/actions/file_manipulation.rb:124:in `template'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/devcontainer/devcontainer_generator.rb:34:in `create_devcontainer'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/command.rb:28:in `run'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `block in invoke_all'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `each'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `map'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `invoke_all'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/app/app_generator.rb:284:in `devcontainer'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/app_base.rb:167:in `public_send'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/app_base.rb:167:in `build'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/generators/rails/app/app_generator.rb:472:in `create_devcontainer_files'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/command.rb:28:in `run'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `block in invoke_all'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `each'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `map'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:134:in `invoke_all'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/group.rb:243:in `dispatch'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/base.rb:584:in `start'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/commands/application/application_command.rb:28:in `perform'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/command.rb:28:in `run'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/command/base.rb:178:in `invoke_command'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/thor-1.3.2/lib/thor.rb:538:in `dispatch'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/command/base.rb:73:in `perform'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/command.rb:71:in `block in invoke'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/command.rb:149:in `with_argv'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/command.rb:69:in `invoke'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/lib/rails/cli.rb:20:in `<top (required)>'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/railties-7.2.1.2/exe/rails:10:in `<top (required)>'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/bin/rails:25:in `load'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/bin/rails:25:in `<top (required)>'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/cli/exec.rb:58:in `load'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/cli/exec.rb:58:in `kernel_load'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/cli/exec.rb:23:in `run'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/cli.rb:455:in `exec'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor.rb:527:in `dispatch'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/cli.rb:35:in `dispatch'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/cli.rb:29:in `start'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/bundler-2.5.22/exe/bundle:28:in `block in <top (required)>'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/site_ruby/3.3.0/bundler/friendly_errors.rb:117:in `with_friendly_errors'
        from /home/vscode/.rbenv/versions/3.3.4/lib/ruby/gems/3.3.0/gems/bundler-2.5.22/exe/bundle:20:in `<top (required)>'
        from /home/vscode/.rbenv/versions/3.3.4/bin/bundle:25:in `load'
        from /home/vscode/.rbenv/versions/3.3.4/bin/bundle:25:in `<main>'
```
### System configuration
**Rails version**:
Rails 7.2.1.2, also happens at edge version
**Ruby version**:
ruby 3.3.4 (2024-07-09 revision be1089c8ec) [aarch64-linux]

**Comment by plattenschieber** [comment]
If my proposed change is viable to you, I'm happy to open a PR and update relevant tests.

**Comment by jrodden1** [comment]
@plattenschieber It looks like this has resurfaced in Rails 8.   

I am getting the same error when using Rails 8.0.2 with ruby 3.4.2 and it leaves behind a `.devcontainer` in the `cwd` in which the generator was run.  The left behind `.devcontainer` directory contains an empty `devcontainer.json` file (like you mentioned in your [PR](https://github.com/rails/rails/pull/53733#issue-2691406810).

I'm a bit new to filing an issue on the `rails` repo.  Should a new issue be opened or should this one be reopened? I'm happy to conform any guidelines for bug submissions (feel free to point me to any docs explaining the correct protocol / next steps).  

## Steps to reproduce
`rails new test-app --devcontainer --pretend`

## Expected behavior
It should run through as it does when leaving out the --pretend flag.

## Actual Results

```
$ rails --version
Rails 8.0.2

$ ruby -v
ruby 3.4.2 (2025-02-15 revision d2930f8e7a) +PRISM [arm64-darwin23]

$ rails new test-app --pretend --devcontainer
      create
      create  README.md
      create  Rakefile
      create  .ruby-version
      create  config.ru
      create  .gitignore
      create  .gitattributes
      create  Gemfile
      [...]
      create  Dockerfile
      create  .dockerignore
      create  bin/docker-entrypoint
      [...]
      create  tmp/storage/.keep
       exist  .devcontainer
/Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in 'block in Thor::Actions#template': undefined method 'as_json' for an instance of Hash (NoMethodError)
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in 'Hash#each'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in 'Enumerable#map'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/devcontainer/templates/devcontainer/devcontainer.json.tt:11:in 'Thor::Actions#template'
	from /Users/myself/.rvm/rubies/ruby-3.4.2/lib/ruby/3.4.0/erb.rb:429:in 'Kernel#eval'
	from /Users/myself/.rvm/rubies/ruby-3.4.2/lib/ruby/3.4.0/erb.rb:429:in 'ERB#result'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/file_manipulation.rb:128:in 'block in Thor::Actions#template'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/create_file.rb:54:in 'Thor::Actions::CreateFile#render'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/create_file.rb:47:in 'Thor::Actions::CreateFile#identical?'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/create_file.rb:74:in 'Thor::Actions::CreateFile#on_conflict_behavior'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/empty_directory.rb:115:in 'Thor::Actions::EmptyDirectory#invoke_with_conflict_check'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/create_file.rb:61:in 'Thor::Actions::CreateFile#invoke!'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions.rb:93:in 'Thor::Actions#action'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/create_file.rb:25:in 'Thor::Actions#create_file'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/actions/file_manipulation.rb:124:in 'Thor::Actions#template'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/devcontainer/devcontainer_generator.rb:40:in 'Rails::Generators::DevcontainerGenerator#create_devcontainer'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/command.rb:28:in 'Thor::Command#run'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:127:in 'Thor::Invocation#invoke_command'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'block in Thor::Invocation#invoke_all'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'Hash#each'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'Enumerable#map'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'Thor::Invocation#invoke_all'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/app/app_generator.rb:278:in 'Rails::AppBuilder#devcontainer'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/app_base.rb:171:in 'Kernel#public_send'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/app_base.rb:171:in 'Rails::Generators::AppBase#build'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/generators/rails/app/app_generator.rb:474:in 'Rails::Generators::AppGenerator#create_devcontainer_files'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/command.rb:28:in 'Thor::Command#run'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:127:in 'Thor::Invocation#invoke_command'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'block in Thor::Invocation#invoke_all'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'Hash#each'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'Enumerable#map'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:134:in 'Thor::Invocation#invoke_all'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/group.rb:243:in 'Thor::Group.dispatch'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/base.rb:584:in 'Thor::Base::ClassMethods#start'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/commands/application/application_command.rb:28:in 'Rails::Command::ApplicationCommand#perform'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/command.rb:28:in 'Thor::Command#run'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor/invocation.rb:127:in 'Thor::Invocation#invoke_command'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/command/base.rb:178:in 'Rails::Command::Base#invoke_command'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/thor-1.3.2/lib/thor.rb:538:in 'Thor.dispatch'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/command/base.rb:73:in 'Rails::Command::Base.perform'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/command.rb:65:in 'block in Rails::Command.invoke'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/command.rb:143:in 'Rails::Command.with_argv'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/command.rb:63:in 'Rails::Command.invoke'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/lib/rails/cli.rb:20:in '<top (required)>'
	from <internal:/Users/myself/.rvm/rubies/ruby-3.4.2/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from <internal:/Users/myself/.rvm/rubies/ruby-3.4.2/lib/ruby/3.4.0/rubygems/core_ext/kernel_require.rb>:136:in 'Kernel#require'
	from /Users/myself/.rvm/gems/ruby-3.4.2/gems/railties-8.0.2/exe/rails:10:in '<top (required)>'
	from /Users/myself/.rvm/gems/ruby-3.4.2/bin/rails:25:in 'Kernel#load'
	from /Users/myself/.rvm/gems/ruby-3.4.2/bin/rails:25:in '<main>'
	from /Users/myself/.rvm/gems/ruby-3.4.2/bin/ruby_executable_hooks:22:in 'Kernel#eval'
	from /Users/myself/.rvm/gems/ruby-3.4.2/bin/ruby_executable_hooks:22:in '<main>'
```

## System Config 
* MBP M1 Max 💻 
* `rvm` to manage ruby versions
* ruby `3.4.2`
* rails `8.0.2`

**Comment by skipkayhil** [comment]
It looks like the fix was not backported to 8-0-stable.

cc @byroot

**Comment by byroot** [comment]
Done.

**Comment by jrodden1** [comment]
Cool.  Thanks Everyone!  

Will this fix go out in 8.0.3 then?
